console/console-init/ui/src/modules/project-detail/components/AccessCredentials/AccessCredentials.tsx

```diff
@@ -119,7 +120,8 @@ const AccessCredentials: React.FunctionComponent<IAccessCredentialsProps> = ({
             value={username}
             isReadOnly={true}
             enableCopy={true}
-            ariaLabel={"input-messaging-username"}
+            id={"messaging-username-input"}
+            ariaLabel={"messaging-username"}
```

Comment:
It should be something like ariaLabel={"messaging username input field"}

Reply:
Fixed